SPECS/kernel/CVE-2010-4563.nopatch

```diff
@@ -0,0 +1 @@
+CVE-2010-4563 - No upstream fix. Low security impact. No plans to fix.
```

Comment:
I usually see comments for nopatch files explaining why it's already fixed or not applicable, but this seems different.  Is nopatch correct for this case?  Wouldn't we want the CVE scanner to still call out if we are vulnerable on this?

Reply:
Agreed. I have adjusted the wording